ssg/build_cpe.py

```diff
@@ -307,6 +313,18 @@ def extract_env_obj(objects, local_var):
     return None
 
 
+def get_correct_elementtree_iterator(tree):
```

Comment:
Does it really matter nowadays? RHEL7 has Python 2.7 (where getiterator was deprecated). Maybe it's a good idea to get rid of it completely?

Reply:
Fixed in commit a21fed4fbc9106425bf8a2f9e36a4ae0af4ee3d7